### ISSUE TITLE
Issue about fuction solver.cal_loss

### ISSUE THREAD

**Issue by poor1017** [jjery2243542/voice_conversion#22]
Hi @jjery2243542 

    I found in function solver.cal_loss:

'logits' dimention is: [batch, n_spkers * enc_len]
'y_true' dimention is: [batch, n_spkers]

Why this mimatch can work well?

Thank you

**Comment by poor1017** [comment]
Because I changed the 'seg len'.
